src/test/java/org/apache/commons/lang3/time/DateUtilsTest.java

```diff
@@ -693,6 +693,59 @@ public void testToCalendar() {
             // expected
         }
     }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateNotNull() {
+        assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the Date back", date1, DateUtils.toCalendar(date1, zone).getTime());
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithDateNull() {
+        try {
+            DateUtils.toCalendar(null, zone);
+            fail("Expected NullPointerException to be thrown when Date is null");
+        } catch(final NullPointerException npe) {
+            // expected
+        }
+    }
+    
+    //-----------------------------------------------------------------------
+    @Test
+    public void testToCalendarWithTimeZoneNotNull() {
+    	assertEquals("Convert Date and TimeZone to a Calendar, but failed to get the TimeZone back", zone, DateUtils.toCalendar(date1, zone).getTimeZone());
```

Comment:
This test is redundant because we have `testToCalendarWithDateAndTimeZoneNotNull()`